products/extensions/subscriptions/guides/mixed-checkout.md

```diff
@@ -0,0 +1,164 @@
+---
+nav:
+  title: Mixed checkout
+  position: 20
+
+---
+# Mixed subscription checkout
+
+::: info
+Available since Shopware version 6.7.4.0
+:::
+
+This guide describes how the mixed carts in the subscription feature works and how extensions should integrate with it.
+Mixed carts let customers buy subscription products and one‑time products in the same checkout while keeping subscription calculation isolated and predictable.
+
+Please familiarise yourself with the [concept](../concept.md) first before continuing here.
+
```

Comment:
I'd love a short paragraph here or after the overview, why mixed carts are such an issue with Shopware and what difficulties we had to overcome to make it work with rules etc, while being performant and not rewriting the whole cart logic. 

I think it could help 3rd party devs to understand, why we use managed carts.